recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div
+    class="dt-attachment-carousel"
+    role="presentation"
+    @mouseenter="showCarouselArrows = true"
+    @mouseleave="showCarouselArrows = false"
+    @focusin="showCarouselArrows = true"
+    @focusout="showCarouselArrows = false"
+  >
+    <div
+      v-if="mediaList.length > 0"
+      ref="carousel"
+      class="dt-attachment-carousel--media-list"
+      :aria-label="attachmentAriaLabel"
+      @scroll="handleScroll"
+    >
```

Comment:
accessbiility warning here

```
.dt-attachment-carousel--media-list

Serious
aria-label attribute is not well supported on a div with no valid role attribute.
```

is there a better semantic element or role?

Reply:
i just put this on the parent one since it has a role presentation. but theres accessbility issue on progress bar am gonna fix.